src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -391,12 +391,14 @@ def setupProcessingTask(self, procTask, taskType, inputDataset=None, inputStep=N
         procTaskCmsswHelper.setOverrideCatalog(self.overrideCatalog)
         procTaskCmsswHelper.setErrorDestinationStep(stepName=procTaskLogArch.name())
 
-        if forceMerged:
+        if forceMerged and isinstance(forceMerged, bool):
             procTaskStageHelper.setMinMergeSize(0, 0)
-        elif forceUnmerged:
+        elif forceUnmerged and isinstance(forceUnmerged, bool):
             procTaskStageHelper.disableStraightToMerge()
         else:
             procTaskStageHelper.setMinMergeSize(self.minMergeSize, self.maxMergeEvents)
+            if forceUnmerged and isinstance(forceUnmerged, list):
```

Comment:
You have one processing step here with multiple ouputs (RECO, AOD, MINIAOD, DQMIO, ALCARECO). All output modules should use normal straight to merge limits, *except* the ALCARECO output which should always be written unmerged.